File: app/src/main/java/ua/te/hackathon/smartcity2015/api/SmartCityService.java
package ua.te.hackathon.smartcity2015.api;

import java.util.List;

import retrofit2.http.GET;
import rx.Observable;
import ua.te.hackathon.smartcity2015.api.model.Event;

/**
 * @author victor
 * @since 2016-02-13
 */
public interface SmartCityService {

  @GET("events/upcoming")
  Observable<List<Event>> getUpcomingEvents();

}

File: app/src/main/java/ua/te/hackathon/smartcity2015/api/SmartCityService.kt
package ua.te.hackathon.smartcity2015.api

import rx.Observable
import ua.te.hackathon.smartcity2015.api.model.Event

/**
 * @author victor
 * *
 * @since 2016-02-13
 */
interface SmartCityService {

  fun upcomingEvents(): Observable<List<Event>>

}
